src/util/strip.ts

```diff
@@ -12,6 +12,10 @@ type StripOptions = { preserveFormatting?: boolean; preventTrim?: boolean; strip
 
 const regexSpanTagOnlyContainsWhitespaces = /<span[^>]*>([\s]+)<\/span>/gim
 
+const regexExpForEmptyFormattingTags = /<[^/>][^>]*>\s*<\/[^>]+>/gim
+
+const regexExpForWhiteSpacesInTags = /[\s]+/gm
```

Comment:
Regex to replace trailing/leading multiline whitespaces with space. The reason for replacing here with space is to respect certain scenarios like `Hello <b>world</b>` with inline spaces.